lib/storage/data/external/AwsClient.js

```diff
@@ -63,9 +68,18 @@ class AwsClient {
                         region,
                         endpoint,
                     });
-                    this._client.endpoint = new AWS.Endpoint(endpoint);
+                    this._client = new S3Client({ ...this._s3Params, endpoint });
```

Comment:
I think it needs region ?
```suggestion
                    this._client = new S3Client({ ...this._s3Params, region, endpoint });
```